src/Tools/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/SyntaxVisualizerControl.xaml.cs

```diff
@@ -35,6 +35,8 @@ private class SyntaxTag
 
         #region Private State
         private TreeViewItem _currentSelection;
+        private Brush _previousForeground;
```

Comment:
This seems fishy that we're having to muck around with this in the first place. My gut is we're missing some XAML declaration somewhere that can do this declaratively.

What's the root cause of this? Why is this the best approach?